capa/features/extractors/ida/insn.py

```diff
@@ -144,12 +144,13 @@ def extract_insn_offset_features(f, bb, insn):
             continue
         p_info = capa.features.extractors.ida.helpers.get_op_phrase_info(op)
         op_off = p_info.get("offset", 0)
-        if 0 == op_off:
-            continue
         if idaapi.is_mapped(op_off):
             # Ignore:
             #   mov esi, dword_1005B148[esi]
             continue
+
+        op_off = capa.features.extractors.helpers.twos_complement(op_off, 32)
```

Comment:
What about 8 and 16?

Reply:
or 64?

Reply:
i got the impression that IDA returns the offset as twos complement in a u32, never as u8/u16/u64. if this is wrong, i'd very much like to fix it.

according to some sources, like [here](https://stackoverflow.com/q/31853189/87207), there is no 64-bit offset/displacement.